### PR TITLE
Use tox-min-req for min-req tests

### PR DESCRIPTION
**Pull request by Czaki**
# Fixes/Closes

# Description

The current workflow for testing min req requires rewriting `setup.cfg` to pin dependencies. This is problematic when adding any `setup.cfg` changes. It also carries the risk of committing such changes to a branch. As a result, many contributors may prefer to depend on CI for min-req testing.

This PR instead uses https://github.com/czaki/tox-min-req a [tox](https://tox.wiki/) plugin that simplifies minimum requirements testing.

With this change, it is enough to call `MIN_REQ=1 tox -e py38-linux-pyqt5` to run Linux min-req tests.

# References
See https://github.com/Czaki/tox-min-req for info on this tox plugin

## Type of change

- [x] Maintenance

# How has this been tested?

- [x] example: all tests pass with my change

## Final checklist:
- [x] My PR is the minimum possible work for the desired functionality

### CODE REVIEW

**Comment by melissawm** [comment]
Rendered docs are visible here, thanks @Czaki !

**Comment by psobolewskiPhD** [comment]
What is up with these CI messages?
image

File: .github/workflows/test_pull_requests.yml [hunk 1]
```diff
@@ -127,8 +127,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install --upgrade pip
-          pip install setuptools tox tox-gh-actions
-          python tools/minreq.py
+          pip install setuptools tox tox-gh-actions tox-min-req
         env:
           # tools/minreq.py sets all deps to their minumim stated versions
```
Comment (psobolewskiPhD) [inline .github/workflows/test_pull_requests.yml]:
```suggestion
          # tools/minreq.py sets all deps to their minumim stated versions
```
This file is deleted in this PR, so probably this comment should be tweaked?

Comment (Czaki) [reply .github/workflows/test_pull_requests.yml]: Yes. good point

**Review by psobolewskiPhD — APPROVED** [review]
Can you clarify what you mean by
>Exposing this way of running min-req test should simplify run such test locally to debug test failure.

?

**Comment by Czaki** [comment]
The current workflow for testing min req requires a rewrite `setup.cfg` for pin dependencies. It is problematic when adding any `setup.cfg` changes. It also provides a risk of committing such files to a branch. So many contributors may depend on CI for min req test. 

With this change, it is enough to call `MIN_REQ=1 tox -e py38-linux-pyqt5` to run Linux min req tests.

**Comment by psobolewskiPhD** [comment]
Do you want to paste that into the PR description?

**Comment by Czaki** [comment]
it si ok now?

**Comment by psobolewskiPhD** [comment]
@Czaki I copy edited the OP and title. What do you think?

**Review by jni — APPROVED** [review]

**Pull request merged**